### DISCUSSION TITLE
Zotero

### DISCUSSION THREAD

**Discussion by sahana-dhar** [Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022#37] [General]
I moved a research article into Zotero, however whenever I try to open it on Zotero only the abstract comes up. I can still write my summary using the abstract and full article on SciHub, but will my citation for the abstract be the same as the full paper?

**Comment by madelyn-redick** [comment]
My guess would be that the citation would be the same. Zotero's default is probably just to open the paper from the original publisher, but since you would need to pay for it you can only see the abstract. But since the DOI is likely the same, the citation will also likely be the same.

**Reply by jonmatthis** [reply]
Yeah, @madelynredick is probably right!

You can also just take a look at the citation that gets produced - if you compare it to another citation (of the same format) you should be able to tell if there are any pieces missing

**Comment by andreabuitrago1** [comment]
You could try comparing it to the citation given by PubMed (if your paper is on there)
